index.js

```diff
@@ -33,7 +33,10 @@ const defaultRules = [
   '.DS_Store',
   '._*',
   '*.orig',
-  'package-lock.json'
+  'package-lock.json',
+  'core',
```

Comment:
I'm worried about this one being a bit too general. Are core dumps seriously just called `core`? 😱

Reply:
@zkat That’s the default at least for Linux (although it’s configurable), and [Wikipedia](https://en.wikipedia.org/wiki/Core_dump#Naming) agrees by calling it the “traditional” naming of core dumps.

It’s also practical – some people configure their core dump names to have the PID or something like that in them, but that isn’t a sensible default because it can mean your machine filling up with core dumps. Just overwriting a file named `core` every time makes quite a bit of sense…

Reply:
On OS X and other BSD-family Unixes, it's common for core dumps to be in `/var/cores` or `/cores`, with a name of `core.${pid}`.  But in those cases, they're not in cwd, so it's less of an issue for this case.

I'm personally fine with ignoring `core` by default, as long as it can be explicitly included by a later rule, but it'd be good to see how often this happens empirically by looking at the registry, and how many files named `core` are not actually core dumps.

Reply:
Oh, actually, I think some versions of SmartOS drop the core file as `core.${pid}` in cwd?  Might be worth also adding `core.+([0-9])` as well.

Reply:
@isaacs yup, done! :+1:

Reply:
[Some people](https://npm.community/t/npm-pack-leaving-out-files-6-8-0-only/5382) do encounter problems with this. Is there any way to make sure directories named `core` don't get removed? Something like `core/` + `**/core/` should work (the latter because [just `core/` doesn't work in subdirectories](https://npm.community/t/gitignore-is-not-working-properly/5330)), but maybe there's a better way.

Reply:
@larsgw Yeah, that makes sense and I don’t see a better way off the top of my head either. Do you want to open a PR for that?

Reply:
Here you go https://github.com/npm/npm-packlist/pull/25

Reply:
I have a "common" utils repo required through git in almost all my microservices, (basically, types and some classes to ease the communication between them). Suddenly, all my classes and types from the "core" service disappeared in my services.

I think this behaviour should be undone as it breaks functional code, and is included in a minor update.